features/app-launch-handler.yml

```diff
@@ -0,0 +1,13 @@
+name: Launch handlers
+description: The `launch_handler` and `client_mode` web app manifest members configure how web app launches should behave when an instance of the app is already open. For example, you can choose to focus on an existing app instead of launching a new one.
```

Comment:
```suggestion
description: The `launch_handler` web app manifest member, with the `client_mode` property, configure how web app launches behave when an instance of the app is already open. For example, you can choose to focus the existing app instance instead of launching a new one.
```